plugins/vst_base/VstPlugin.cpp

```diff
@@ -629,6 +568,60 @@ void VstPlugin::idleUpdate()
 	unlock();
 }
 
+void VstPlugin::showUI()
+{
+	QString embedMethod = ConfigManager::inst()->vstEmbedMethod();
+	if ( embedMethod == "none" )
+	{
+		RemotePlugin::showUI();
+	}
+	else
+	{
+		if (! pluginWidget()) {
+			createUI( NULL, false );
+		}
+
+		QWidget * w = pluginWidget();
+		if( w )
+		{
+#		ifdef LMMS_BUILD_WIN32
+			// hide sw, plugin window wrapper on win32
+			// this is obtained from pluginWidget()
+			if( isEffect )
```

Comment:
Here `isEffect` is undefined. I think you should store it from `createUI()` or somehow drop it if possible.